src/client/components/pages/parts/revisions-table.js

```diff
@@ -28,67 +28,115 @@ const {formatDate} = utilsHelper;
+				<h1 className="text-center">{tableHeading}</h1>
 			</div>
 			<hr className="thin"/>
-			<Table
-				responsive
-				className={tableCssClasses}
-			>
-				<thead>
-					<tr>
-						<th className="col-sm-2">Revision ID</th>
-						<th className="col-sm-5">Modified entities</th>
-						<th className="col-sm-3">User</th>
-						<th className="col-sm-2">Date</th>
-
-					</tr>
-				</thead>
-				<tbody>
-					{
-						results.map((revision, i) => (
-							<tr key={i}>
-								<td>
-									<a href={`/revision/${revision.revisionId}`} >
-										{revision.revisionId}
-									</a>
-								</td>
-								<td>
-									{revision.entities.map(entity => (
-										<div key={`${revision.revisionId}-${entity.bbid}`}>
-											<a href={getEntityUrl(entity)} >
-												{genEntityIconHTMLElement(entity.type)}
-												{getEntityLabel(entity)}
-											</a>
-										</div>
-									))}
-								</td>
-								<td>
-									<a href={`/editor/${revision.editor.id}`} >
-										{revision.editor.name}
-									</a>
-								</td>
-								<td>{formatDate(new Date(revision.createdAt), true)}</td>
+			{
+				results.length > 0 ?
+					<Table
+						responsive
+						className={tableCssClasses}
+					>
+						<thead>
+							<tr>
+								<th className="col-sm-2">Revision ID</th>
+								<th className="col-sm-5">Modified entities</th>
+								{
+									showRevisionEditor ?
+										<th className="col-sm-3">User</th> : null
+								}
+								{
+									showRevisionNote ?
+										<th className="col-sm-3">Note</th> : null
+								}
+								<th className="col-sm-2">Date</th>
 							</tr>
-						))
-					}
-				</tbody>
-			</Table>
+						</thead>
+
+						<tbody>
+							{
+								results.map((revision, i) => (
+									<tr key={i}>
+										<td>
+											<a href={`/revision/${revision.revisionId}`} >
+												{revision.revisionId}
+											</a>
+										</td>
+										<td>
+											{revision.entities.map(entity => (
+												<div key={`${revision.revisionId}-${entity.bbid}`}>
+													<a href={getEntityUrl(entity)} >
+														{genEntityIconHTMLElement(entity.type)}
+														{getEntityLabel(entity)}
+													</a>
+												</div>
+											))}
+										</td>
+										{
+											showRevisionEditor ?
+												<td>
+													<a href={`/editor/${revision.editor.id}`} >
+														{revision.editor.name}
+													</a>
+												</td> : null
+										}
+										{
+											showRevisionNote ?
+												<td>
+													{revision.notes.map(note => (
+														<div key={note.id}>
+															{/* eslint-disable-next-line react/no-unescaped-entities */}
+															"{note.content}"
+															<a className="pull-right" href={`/editor/${note.author.id}`}>
+																<i>
+																	—{note.author.name}
+																</i>
+															</a>
+															<br/>
+															<br/>
```

Comment:
I like this thin line between notes. Otherwise in some cases it was getting difficult to distinguish b/w two notes. That is why I added `<br/>` . and yeah it was creating an empty line in some cases, I should've tested it properly.
I don't know much css, I was struggling with it. Thanks :D
Also I personally would've liked if note-author is at the right side. Especially for the second note in this picture. What do you think?

Reply:
I was wondering how that would work, and I was curious so i had a go :)

Here's the markup and CSS I ended up with:
```
<div className="revision-note" key={note.id}>
	<span className="note-content">
		{note.content}
		<a
			className="note-author pull-right" href={`/editor/${note.author.id}`}
		>
			—{note.author.name}
		</a>
	</span>
</div>
```

```
.revision-note {
	&:not(:last-child) {
		border-bottom: 1px solid @disabled-color;
		margin-bottom: 0.3em;
		padding-bottom: 0.3em;
	}
	.note-content {
		quotes: "\201C" "\201D";
		&::before {
				content: open-quote;
		}
		&::after {
				content: close-quote;
		}
	}
	.note-author {
		&:extend(.small);
		font-style: italic;
		white-space: nowrap;
		margin-left: 0.2em;
	}
}
```